packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -59,6 +65,7 @@ export class DataBuilder {
     };
 
     return this.aggregator.fetchAnalytics(dataElementCodes, fetchOptions, this.query, {
+      aggregations,
```

Comment:
This could break existing functionality. I checked existing reports,overlays, data and measure builders and none of them seems to have an `aggregation` field in their config - so I guess we're fine!

`aggregations` will be used if defined, if not, `aggregationType`/`aggregationConfig` - see [buildAggregationOptions](https://github.com/beyondessential/tupaia/blob/dev/packages/web-config-server/src/aggregator/buildAggregationOptions.js#L32)

We should certainly refactor how we handle aggregation in data builders - it's highly inconsistent!